lib/src/exceptions/logic/signal_width_mismatch_exception.dart

```diff
@@ -31,4 +31,12 @@ class SignalWidthMismatchException extends RohdException {
   SignalWidthMismatchException.forNull(dynamic val)
       : super('Could not infer width of input $val.'
             ' Please provide a valid width.');
+
+  /// Constructs a new [Exception] for when a dynamic has a wrong width.
+  SignalWidthMismatchException.forWidthOverflow(int actualWidth, int maxWidth,
+      {String customMessage = ''})
```

Comment:
You could make it `String? customMessage` so that you can do `customMessage ?? 'Value has...'` instead of checking if it's an empty string